VultisigApp/VultisigApp/Views/GeneralCodeScannerView.swift

```diff
@@ -114,28 +118,59 @@ struct GeneralCodeScannerView: View {
         case .SignTransaction:
             moveToVaultsView()
         case .Unknown:
-            return
+            moveToSendView()
         }
     }
     
     private func moveToCreateVaultView() {
-        shouldJoinKeygen = true
+        shouldSendCrypto = false
         showSheet = false
+        shouldJoinKeygen = true
     }
     
     private func moveToVaultsView() {
         DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
-            shouldKeysignTransaction = true
             showSheet = false
+            shouldSendCrypto = false
+            shouldKeysignTransaction = true
+        }
+    }
+    
+    private func moveToSendView() {
+        shouldJoinKeygen = false
+        showSheet = false
+        checkForAddress()
+    }
+    
+    private func checkForAddress() {
+        let address = deeplinkViewModel.address ?? ""
+        sendTX.toAddress = address
+        
+        for asset in settingsDefaultChainViewModel.baseChains.sorted(by: {
```

Comment:
it is better for performance if you set the sorted value in a variable before interact with it. 

like 

```
let sortedAssets = settingsDefaultChainViewModel.baseChains.sorted ...
```